scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -33,13 +33,92 @@ async function main() {
   const sdl = (await ethers.getContract("SDL")) as SDL
   const veSDL = (await ethers.getContract("VotingEscrow")) as VotingEscrow
   const minter = (await ethers.getContract("Minter")) as Minter
-  const DAY = 86400
   const WEEK = 86400 * 7
   const YEAR = WEEK * 52
-  const GaugeHelperContract = (await ethers.getContract(
+  const MAXTIME = 86400 * 365 * 4
+  const gaugeHelperContract = (await ethers.getContract(
     "GaugeHelperContract",
   )) as GaugeHelperContract
-  //  Deploy dummy reward tokens
+
+  const numberOfGauges = (await gaugeController.n_gauges()).toNumber()
+  const gauges: { name: string; address: string }[] = []
+
+  for (let i = 0; i < numberOfGauges; i++) {
+    const address = await gaugeController.gauges(i)
+    const gaugeContract = (await ethers.getContractAt(
+      "LiquidityGaugeV5",
+      address,
+    )) as LiquidityGaugeV5
+
+    let name: string
+    try {
+      name = await gaugeContract.name()
+    } catch (e) {
+      // In case of root gauges, they don't have a name.
+      name = address
+    }
+
+    gauges.push({ name, address })
+  }
+
+  // The deploy scripts should have already added a default gauge type
+  expect((await gaugeController.n_gauge_types()).toNumber()).to.eq(2)
```

Comment:
may be untrue if we're not deploying rootGauges

Reply:
you are correct, changing this to 1 is our check that root gauges where not added to gaugeHelper